File: Python/Medium/Stack/2390. Removing Stars From a String.py
def removeStars(word: str) -> str:
    stack = []
    for i in word:
        if i == '*':
            stack.pop()
        else:
            stack.append(i)
    return ''.join(stack)


word = "leet**cod*e"
print(removeStars(word))

File: PHP/Medium/Stack/2390_Removing_Stars_From_a_String.php
<?php
//https://leetcode.com/problems/removing-stars-from-a-string/description/

function removeStars($s)
{
    $stack = [];
    foreach (str_split($s) as $char) {
        if ($char == '*')
            array_pop($stack);
        else {
            array_push($stack, $char);
        }
    }
    return join($stack);
}

$s = "leet**cod*e";
$s = "erase*****";
echo removeStars($s);